syntax = "proto3";
package object_store;




import "google/protobuf/empty.proto";

/*
RemoveObject请求
*/
message RemoveObjectRequest {
    /*
    存储桶名称
    */
    string bucketName = 1 ;
    /*
    存储桶中对象名称
    */
    string objectName = 2 ;
}

/*
RemoveObjectApi返回
*/
message RemoveObjectResponseWrapper {
    /*
    返回码
    */
    int32 code = 1 ;
    /*
    返回码解释
    */
    string codeExplain = 2 ;
    /*
    错误详情
    */
    string error = 3 ;
    /*
    返回数据
    */
    google.protobuf.Empty data = 4 ;
}

